public/app/features/admin/state/reducers.ts

```diff
@@ -201,8 +203,60 @@ export const { usersFetched, usersFetchBegin, usersFetchEnd, queryChanged, pageC
   userListAdminSlice.actions;
 export const userListAdminReducer = userListAdminSlice.reducer;
 
+// UserListAnonymousPage
+
+const initialUserListAnonymousState: UserListAnonymousState = {
+  devices: [],
+  query: '',
+  page: 0,
+  perPage: 50,
+  totalPages: 1,
+  showPaging: false,
+  filters: [{ name: 'activeLast30Days', value: false }],
```

Comment:
<img width="1092" alt="image" src="https://github.com/grafana/grafana/assets/7727602/974929b4-fb88-4bf0-b971-0a91dc36748b">
i have it there for a placeholder, as we are not able to filter/search users currently. For that we need to make a query endpoint for the devices and that is not in scope currently.

I'd rather keep these as they are needed for the interactive table and the users table. If it is a must to remove, I could look into it.

Reply:
The filters are not necessary for the interactive table, you can remove those and all the related code from the `UserListAnonymousPage`. In fact, I'd just remove all the code not in scope for this PR, it can be added later if needed.